molgenis-app/development/docker-compose.yml

```diff
@@ -0,0 +1,55 @@
+version: "3.4"
+services:
+  frontend:
+    image: ${FRONTEND}
+    ports:
+      - 80:80
+    volumes:
+      - ${BACKEND}:/etc/nginx/proxy.d/backend.conf
+
+  postgres:
```

Comment:
How should I now connect to it in the terminal? Just using `psql` gives:

```
psql: could not connect to server: No such file or directory
        Is the server running locally and accepting
        connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
```

Reply:
Check DOCKER.md.